README.md

```diff
@@ -127,6 +127,8 @@ All tests are available under [src/test/java](https://github.com/temporalio/samp
 
 - [**Customer Application Approval DSL**](https://github.com/temporalio/samples-java/tree/master/src/main/java/io/temporal/samples/dsl): Demonstrates execution of a customer application approval workflow defined in a DSL (like JSON or YAML)
 
+- [**Polling Services**](https://github.com/temporalio/samples-java/tree/master/src/main/java/io/temporal/samples/polling): Demonstrates best practices for polling services
```

Comment:
I would probably change the wording. It's hardly can be called "best practice", because it's elegant, but a hack.
Maybe we should call it "a technique"?